werkomgeving/gegeven/index.md

```diff
@@ -20,29 +20,29 @@ In bovenstaande voorbeelden wordt met [Jan] de verwijzing bedoeld naar een speci
 
 ## Een gegevenstype
 
-Een gegevenstype is *een typering van gelijksoortige gegevens*. Met een gegevenstype beschrijven we hoe een afzonderlijk gegeven er uit ziet, waarbij bovenstaande vier elementen terugkomen. Van een gegevenstype wil je de volgende zaken weten:
+Een gegevenstype is *een typering van gelijksoortige gegevens*. Met een gegevenstype beschrijven we hoe een afzonderlijk gegeven er uit ziet, waarbij bovenstaande vier elementen terugkomen. Van een gegevenstype wil je de volgende zaken weten: 
 
-1. Hoe refereer je naar het object waarover het gegeven gaat? Welke identificerende eigenschappen gebruik je daarbij?
+1. Hoe refereer je naar het object waarover het gegeven gaat? Welke identificerende eigenschappen gebruik je daarbij? 
 2. Over welke eigenschap gaat zo'n gegeven? (dwz: relatie naar het CIM waarin de eigenschappen zijn beschreven)
-3. Welke waarden zijn mogelijk bij dit gegeven, welke beperkingen gelden hierbij? En hoeveel waarden zijn (gelijktijdig) mogelijk voor de betreffende eigenschap?
-4. Welke informatie is er over de uitspraak zelf nodig?
-5. In welke vorm vindt de vastlegging logisch gezien plaats, zijn er beperkingen aan de manier waarop deze vastlegging mag plaatsvinden?
+3. Welke waarden zijn mogelijk bij dit gegeven, welke beperkingen gelden hierbij? En hoeveel waarden zijn (gelijktijdig) mogelijk voor de betreffende eigenschap? 
+4. Welke informatie is er over de uitspraak zelf nodig? 
+5. In welke vorm vindt de vastlegging logisch gezien plaats, zijn er beperkingen aan de manier waarop deze vastlegging mag plaatsvinden? 
 6. Is een uitspraak over deze eigenschap van het object verplicht om vast te leggen?
 
-Vier verschillende soorten gegevenstypen zijn te onderscheiden:
+Vier verschillende soorten gegevenstypen zijn te onderscheiden: 
 
 1. Gegevenstypen waarbij sprake is van *letterlijke waarden*. Een gegevenstype over de eigenschap «leeftijd» of «BSN-nummer» zijn voorbeelden van dergelijke gegevenstypen.
 2. Gegevenstypen waarbij sprake is van *classificerende waarden*. Bij dit gegevenstype is geen sprake van een letterlijke waarde en refereert de waarde aan een betekenis die meer is dan de letterlijke waarde zelf. Een gegevenstype over de eigenschap «geslacht» is een voorbeeld van een dergelijk gegevenstype.
-3. Gegevenstypen waarbij sprake is van *complexe waarden*. Bij dit gegevenstype is sprake van een waarde die is opgebouwd uit enkele afzonderlijke onderdelen die gezamenlijk de complexe waarde vormen. Een gegevenstype over de eigenschap «lengte» is een voorbeeld van een dergelijke gegevenstype (in dit geval bestaat de complexe waarde uit een getal en een eenheid).
```

Comment:
Ik ga er vanuit dat we dit als iets anders zien dan gegevensobjecten. Zo ja, wat is dan precies het argument waardoor we gegeven onder dit type scharen i.p.v. het als een gegevensobject te zien?